### PR TITLE
add levelToLevelCode

### PR DESCRIPTION
**Pull request by andreineculau**
~~EDIT: also adding a `levelToLevelCode` function (needed in aws-util-firecloud)~~

(pushed the no-brainer "naming" diffs to master)

### CODE REVIEW

File: src/listeners/log-to-console.js [hunk 1]
```diff
@@ -53,10 +53,9 @@ export let logToConsole = function(cfg = {}) {
       return;
     }
 
-    let maxLevelName = cfg.level;
-    let maxLevel = logger.levels[maxLevelName];
-    maxLevel = _.floor(maxLevel / 10) * 10 + 10 - 1; // round up to next level, not inclusive
-    if (entry._level > maxLevel) {
+    let maxLevelCode = logger.levelToLevelCode(cfg.level);
+    maxLevelCode = _.floor(maxLevelCode / 10) * 10 + 10 - 1; // round up to next level, not inclusive
```
Comment (IanSavchenko) [inline src/listeners/log-to-console.js:57]: why not `(_.floor(maxLevelCode / 10) + 1) * 10 - 1;` 
or just™ `_.floor(maxLevelCode / 10) * 10 + 9`?

**Review by IanSavchenko — APPROVED** [review]

File: src/listeners/log-to-console.js (hunk 1 above)
Comment (andreineculau) [reply src/listeners/log-to-console.js:57]: or `_.ceil(maxLevelCode / 10) * 10 - 1` maybe

the "- 1" comes the "not inclusive" in the comment.
I cannot think how I'd translate "+ 9" into plain English

Comment (IanSavchenko) [reply src/listeners/log-to-console.js:57]: > or _.ceil(maxLevelCode / 10) * 10 - 1 maybe

won't work for an already rounded level like `60`, which will become 59 as a result

Comment (andreineculau) [reply src/listeners/log-to-console.js:57]: to be continued in https://github.com/tobiipro/minlog/pull/8

**Pull request merged**